.devcontainer/Dockerfile

```
@@ -0,0 +1,13 @@
+FROM golang:1.23-alpine AS builder
```

```suggestion
FROM golang:1.23.7-alpine3.21 AS builder
```
I encountered an error because my local `1.23-alpine` is actually on 1.23.6 when the language server was expecting 1.23.7.